tests/torch/requirements.txt

```diff
@@ -14,4 +14,5 @@ transformers==4.23.1
 # Required for movement_sparsity tests
 datasets==2.6.2
 evaluate==0.3.0
-timm
\ No newline at end of file
+timm==0.6.13
+openvino-dev==2023.0.0.dev20230217
```

Comment:
openvino-dev~=2023.0.0.dev

The last dev or release openvino-dev package will be installed.

Reply:
thanks, corrected